packages/frontend/src/components/dialogs/ProxyConfiguration/styles.module.scss

```diff
@@ -34,6 +30,9 @@
 
 .proxyLabel {
   margin-bottom: 6px;
+  max-width: 320px;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
We still have line breaks.

<img width="434" height="123" alt="Image" src="https://github.com/user-attachments/assets/2600803b-d43d-4325-80df-2e46c6f4b4b8" />

Maybe it's better to either add `white-space: nowrap` or to remove `overflow: hidden` and add `word-break: break-word;`

Reply:
WDYM "can't do anything with that"?
Not really a problem, but maybe can be considered inconsistent. Sometimes we have line breaks, sometimes we truncate.